Few-iOS/TableView.swift

```diff
@@ -104,19 +106,40 @@ public class TableView: Element {
     public override func applyDiff(old: Element, realizedSelf: RealizedElement?) {
         super.applyDiff(old, realizedSelf: realizedSelf)
         
-        if let scrollView = realizedSelf?.view as? FewTableView {
-            let handler = scrollView.handler
+        if let tableView = realizedSelf?.view as? FewTableView {
+            let handler = tableView.handler
             
             layoutElements()
             
             handler?.elements = elements
             
-            let tableView = scrollView.handler?.tableView
-            
+            let newHeaderView = header?.realize(realizedSelf).view
```

Comment:
Is this correct? I'd hate to re-render the header if it hasn't changed but we need to tell the table view whenever it _does_ change.